src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DynamicListHelper.java

```diff
@@ -1,9 +1,13 @@
 package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
 
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
```

Comment:
I think this is the incorrect CollectionUtils package, not sure which specific one we should be using though as i can see both the`org.springframework.util.CollectionUtils` and `org.apache.commons.collections4.CollectionUtils` in the project....

Reply:
The `org.springframework.util.CollectionUtils` does not have `isNotEmpty` whilst the other one does. I could just do `!CollectionUtils.isEmpty` but in previous PRs, I was told to use `isNotEmpty` hence the change in library on this occasion

Reply:
should use `org.apache.commons.collections4.CollectionUtils` as you've done in the other files for this rather than mixing in a new package here.

Reply:
Refactored